package.json

```
@@ -84,7 +82,7 @@
     "proxyquire": "^2.0.1",
     "registry-url": "^5.1.0",
     "sinon": "^6.1.3",
-    "xo": "0.38.0"
+    "xo": "^0.47.0"
```

```suggestion
    "xo": "0.38.0"
```
There are security problems, but this is dev only, won’t be a problem for npm install yo.
Version 0.40.0 is esm only. So 0.39.x should be to newest possible.